### ISSUE TITLE
Bug(sort-switch-case): `case 'default':` is confounded with `default:` clause 

### ISSUE THREAD

**Issue by jerome-benoit** [azat-io/eslint-plugin-perfectionist#229]
### Describe the bug

A switch ... case control flow with a `case 'default` is mishandled and report a false positive error. 

### Code example

```js
    switch (variable) {
      case 'add':
        break
      case 'remove':
        break
      case 'default':
        break
      default:
        break
```

### ESLint version

9.9.0

### ESLint Plugin Perfectionist version

3.2.0

### Additional comments

Using `recommended-natural` configuration. 

### Validations

- [X] Read the [docs](https://perfectionist.dev).
- [X] Check that there isn't [already an issue](https://github.com/azat-io/eslint-plugin-perfectionist/issues) that reports the same bug to avoid creating a duplicate.

**Comment by azat-io** [comment]
Fixed in 5583eb0. Released in [v3.3.0](https://github.com/azat-io/eslint-plugin-perfectionist/releases/tag/v3.3.0).
